src/TaskDetail/TaskDetail.js

```diff
@@ -106,6 +106,30 @@ class TaskDetail  extends React.Component {
     return   this.formatDateToString(millisecs)
   }
 
+  getCommentDisplayText = (commentJSONStr) => {
+
+    commentJSON = JSON.parse(commentJSONStr)
```

Comment:
Hey not putting a let in front of a variable declaration makes it a global. Doesn't really matter here as you always assign commentJSON anyways. But worth noting...